h/tasks/indexer.py

```diff
@@ -23,6 +23,13 @@ def add_annotation(id_):
     search_index.add_annotation_by_id(id_)
 
 
+@celery.task
+def add_annotations_between_times(start_time, end_time, tag):
+    celery.request.find_service(
+        name="search_index"
+    )._queue.add_annotations_between_times(start_time, end_time, tag)
```

Comment:
The new Celery task that actually adds the jobs to the DB

Reply:
I think we're missing test coverage for this. Not much going on, but we do it for the rest. If you added a pass through function on the service, there's a test setup which would allow you to do it with parameterisation alone, but I doubt this is going to be too hard.